File: r/randomforest.r
# Import dataset
df = read.csv('data.csv')

# Create Boolean vector with random split
library(caTools)
split = sample.split(df$y, SplitRatio=0.8)

# Split into training and test set
train = subset(df, split == TRUE)
test = subset(df, split == FALSE)

# Implements Breiman's random forest algorithm.
install.packages('randomForest')
library(randomforest)

# Fit support vector regression to training set
model = randomForest(y ~ ., data=train, ntree=100)

# Predict test set
y_pred = predict(model, newdata=test)

File: python/randomforest.py
import numpy as np
import pandas as pd

# Import dataset
df = pdf.read_csv('data.csv')
X = df.iloc[:, :-1].values
y = df.iloc[:, -1].values

# Split into training and test sets
from sklearn.cross_validation import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)

# Fit support vector regression to training set
from sklearn.ensemble import RandomForestRegressor
model = RandomForestRegressor(n_estimators=100)
model.fit(X_train, y_train)

# Predict test set
y_pred = model.predict(X_test)